chore(blog): add the july community summary (#4287)
asyncapi/website: markdown/blog/2025-july-summary.md

<a href='https://smartbear.com/' target='_blank'>
<img src='/img/sponsors/smartbear_logo.png' alt='SmartBeat' width='500px' />
</a>

[Solace](https://solace.com) as our Gold sponsor.

<a href='https://solace.com' target='_blank'>
<img src='/img/sponsors/solace.png' alt='Solace' width='500px' />
</a>

## Governance Board

We concluded our first [Governance Board](https://www.asyncapi.com/community/board) elections, and we are happy to welcome [Fran Méndez](https://www.linkedin.com/in/fmvilas), [Thulisile Sibanda](https://www.linkedin.com/in/v-thulisile-sibanda), and [Hugo Guerrero](https://www.linkedin.com/in/hugoguerrero) as the first members.

Exciting times are ahead as we grow as a community.

## AsyncAPI Conference 2025
We had two amazing conferences in two new cities, one in Munich, Germany, and the other in Lagos, Nigeria. [APIdays](https://www.apidays.global) and [APIConf Lagos](https://apiconf.net) hosted us as part of the 2-day conference, and we also brought the API Standards Booth.

> ### Munich

Reviewer: ```suggestion
### Munich

We had a 2-day event in Munich, Germany. The most important thing we learned is that AsyncAPI is used in production by insurance giant Allianz, and that the whole event-driven topology at retailers like Lidl and Kaufland (under Schwarz Group) is managed with AsyncAPI.

The first day, what became our tradition, we started with the booth, where we could provide vendor-neutral consulting for the API community. The booth was there under the name `API Standards` with people not only from AsyncAPI but also JSON Schema, OpenAPI, and Arazzo.  In case you want to learn more about the booth, how we organize it, and where, join [API Standards mailing group](https://groups.io/g/api-standards).

----booth image here----

The second day, we had an AsyncAPI track with four amazing presentations. The below short recap will nicely summarize how great it was.

----embed video here----
```

video for embed: https://www.youtube.com/shorts/fsXVL9UUEiQ?feature=share

image from the booth: 

![Image](https://github.com/user-attachments/assets/1ffb625c-7907-425f-b16a-19016f3d75eb)

---
title: "Monthly Community Update: July 2025"
date: 2025-08-04T06:00:00+01:00
type: Communication
tags:
  - Project Status
cover: /img/posts/2025-blog-banner/banner-july.webp
authors:
  - name: Thulisile Sibanda
    photo: /img/avatars/thulieblack.webp
    link: https://www.linkedin.com/in/v-thulisile-sibanda/
    byline: Community Builder and Open Source Fanatic!
excerpt: 'AsyncAPI community status and project updates for July 2025'
featured: true
---
It's been a very busy month in July as we had two conferences in two new cities, and we also had our first elected governance board members. It makes it exciting as we work together towards improving and investing in the growth and sustainability of the AsyncAPI Initiative.

> But before we dive in, I want to acknowledge the continuous support we received from our sponsors in June.

Reviewer 2: A blog post has an author, so that it is clear who speaks and who is referenced as "I".
But who are "we"?

</a>

[Solace](https://solace.com) as our Gold sponsor.

<a href='https://solace.com' target='_blank'>
<img src='/img/sponsors/solace.png' alt='Solace' width='500px' />
</a>

## Governance Board

We concluded our first [Governance Board](https://www.asyncapi.com/community/board) elections, and we are happy to welcome [Fran Méndez](https://www.linkedin.com/in/fmvilas), [Thulisile Sibanda](https://www.linkedin.com/in/v-thulisile-sibanda), and [Hugo Guerrero](https://www.linkedin.com/in/hugoguerrero) as the first members.

Exciting times are ahead as we grow as a community.

## AsyncAPI Conference 2025
We had two amazing conferences in two new cities, one in Munich, Germany, and the other in Lagos, Nigeria. [APIdays](https://www.apidays.global) and [APIConf Lagos](https://apiconf.net) hosted us as part of the 2-day conference, and we also brought the API Standards Booth.

### Munich

We had a 2-day event in Munich, Germany. The most important thing we learned is that AsyncAPI is used in production by insurance giant Allianz, and that the whole event-driven topology at retailers like Lidl and Kaufland (under Schwarz Group) is managed with AsyncAPI.

> The first day, what became our tradition, we started with the booth, where we could provide vendor-neutral consulting for the API community. The booth was there under the name `API Standards` with people not only from AsyncAPI but also JSON Schema, OpenAPI, and Arazzo.  In case you want to learn more about the booth, how we organize it, and where, join the [API Standards mailing group](https://groups.io/g/api-standards).

Reviewer 3: ```suggestion
The first day, what became our tradition, we started with the booth, where we could provide vendor-neutral consulting for the API community. The booth was there under the name `API Standards` with people not only from AsyncAPI but also JSON Schema, OpenAPI, and Arazzo. In case you want to learn more about the booth, how we organize it, and where, join the [API Standards mailing group](https://groups.io/g/api-standards).
```

  src="/img/posts/lagos-conf/booth-munich.webp"
  caption="Booth in Munich From left: Ben, Frank, Lukasz."
  className="text-center"
/>

The second day, we had an AsyncAPI track with four amazing presentations. The below short recap will nicely summarize how great it was.

<YouTube id="fsXVL9UUEiQ" />

### Lagos
For the 2-day event in Lagos, the AsyncAPI track took place on the first day with an amazing lineup of speakers from our community sharing their knowledge and work.

Maya started the track with her talk on how I Came to Design, I Ended Up Leading a Movement, sharing her journey from contributing as a designer to becoming a maintainer.

<Figure
  src="/img/posts/lagos-conf/maya.webp"
  caption="Maya presenting I Came to Design, I Ended Up Leading a Movement."
  className="text-center"
/>

Prince followed with his session on Contributing to AsyncAPI from a Technical Writer's POV, where he spoke on the challenges he faced in contributing to AsyncAPI docs, how he overcame these challenges, and what he learnt.

<Figure
  src="/img/posts/lagos-conf/prince.webp"
  caption="Prince presenting Contributing to AsyncAPI from a Technical Writer's POV."
  className="text-center"
/>

> Ace and Thulie shared a session on An Intro to AsyncAPI Without The Technical Jargon, how the Open Source community strives on collaboration, how we're growing and evolving, and Maintainership.

Reviewer 3: ```suggestion
Ace and Thulie shared a session on 'An Intro to AsyncAPI Without The Technical Jargon', how the Open Source community strives on collaboration, how we're growing and evolving, and Maintainership.
```

  caption="Thulie presenting From Ingredients to Output: The AsyncAPI Generator 101."
  className="text-center"
/>

Ace closed the track by sharing on Preventing API Drift, strategies, and best practices.

<Figure
  src="/img/posts/lagos-conf/ace.webp"
  caption="Azeez presenting Preventing API Drift."
  className="text-center"
/>

#### API Standards Booth

JSON Schema joined us at the booth, and we had many interesting conversations. On the first day, people were eager to learn how two communities can come together to promote Open Source. There was curiosity in the atmosphere as attendees sought to understand how individuals with diverse skill sets collaborate and share their passion for the work being done in the AsyncAPI community.

Day two was even more vibrant. After the track presentations, many attendees had questions and were inspired by the topics discussed. Our maintainers and volunteers did an outstanding job of educating others and answering questions about the community, the specification, and the associated tooling.

<Figure
  src="/img/posts/lagos-conf/booth-duty.webp"
  caption="AsyncAPI Maintainers Answering Questions."
  className="text-center"
/>

> I would like to thank everyone for taking the time to attend the conference and for volunteering at the API Standards Booth.

Reviewer 2: Same here

Reviewer 4: > Same here

I noticed that "we" has been used the most so I implemented the suggestion to use "we" for consistency @bandantonio

Author: Hi Christine so were I used "we" it had a different context to when I used "I". So with blog posts, the most important is to not have a formal tone but informal and relate more with the reader. 
You'll get to see this the more you review different blogs that we write in the community

Reviewer 2: @CBID2 yeah, I got your point, but you need to understand your audience, who are you talking to and who is the person (or people) behind the text. In this case, AsyncAPI has different authors who contribute to blog, so it's natural to talk to the audience (community) from the author's perspective even if this author implies the whole organization (in most cases it's true as well). "We" is always indefinite and vague as it depends on the context. It should be mentioned in the text before, otherwise it is hard to identify what group of people you are referring to as "we". That's why, for example, in technical writing, using "we" is discouraged.

---
title: "Monthly Community Update: July 2025"
date: 2025-08-04T06:00:00+01:00
type: Communication
tags:
  - Project Status
cover: /img/posts/2025-blog-banner/banner-july.webp
authors:
  - name: Thulisile Sibanda
    photo: /img/avatars/thulieblack.webp
    link: https://www.linkedin.com/in/v-thulisile-sibanda/
    byline: Community Builder and Open Source Fanatic!
excerpt: 'AsyncAPI community status and project updates for May 2025'
featured: true
---

## AsyncAPI Conference 2025

> ### Munich

Author: @derberg please add your community summary
